### DISCUSSION TITLE
ClipboardHistory for ClipboardExtension

### DISCUSSION THREAD

**Discussion by lifesign** [espanso/espanso#1191] [Feature requests and Ideas]
i think it will be very useful when clipboardExtension support history, like the same feature in alfred.

syntax may like this
`{{clipboard:0}}`, `{{clipboard:1}}`, `{{cliboard:nth}}`

the number is the order of the clipboard history stack

**Comment by federico-terzi** [comment]
Hey @lifesign,

Thank you for the feedback! That might be a useful feature indeed, so I'll make sure to consider it when working on the next versions

Cheers :)

**Reply by atanasj** [reply]
Has this feature been implemented?
